src/components/nav/Nav.tsx

```diff
@@ -5,15 +5,17 @@ import Image from 'next/image'
 import Link from 'next/link'
 import NavLink from '@/components/nav/NavLink'
 import { useSearch } from '@/components/search/SearchContext'
+import GlobalSearch from '../search/GlobalSearch'
 
 export default function Nav() {
   const { setSearchOpen } = useSearch()
 
   return (
     <header>
+      <GlobalSearch />
       <nav
         className={cn(
-          'mb-[79px] flex w-full items-center px-6 md:mb-[153px] md:px-0'
+          'my-10 mb-10 flex w-full items-center px-6 md:mb-16 md:px-0'
```

Comment:
Why are you using both `my` and `mb` here? Would it make more sense to just use `mt` and `mb`?

Reply:
Removed `mb-10` and updated to `my-10 md:mb-16`. That way, we only change bottom margin on medium and up screens.